sql/src/main/java/io/crate/exceptions/RelationUnknownException.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Licensed to Crate under one or more contributor license agreements.
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.  Crate licenses this file
+ * to you under the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied.  See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial
+ * agreement.
+ */
+
+package io.crate.exceptions;
+
+import javax.annotation.Nullable;
+import java.util.Locale;
+
+public class RelationUnknownException extends ResourceUnknownException {
+
+    private static final String MESSAGE = "Cannot resolve relation '%s.%s'";
+    private static final String MESSAGE_TABLE_ONLY = "Cannot resolve relation '%s'";
+
+    private final String fqn;
+
+    public RelationUnknownException(String tableName, @Nullable String schemaName) {
+        super(schemaName == null ? String.format(Locale.ENGLISH, MESSAGE_TABLE_ONLY, tableName)
```

Comment:
is there always a schema? it would make sense to use a QualifiedName instead of this fqn String, since those names here might be relative or point to aliases. so the whole conversion here in the constructor is not required